paper/paper.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'ReLax: Efficient and Scalable Recourse Explanation Benchmark using JAX'
+tags:
+  - Python
+  - JAX
+  - machine learning
+  - interpretability
+  - counterfactual explanation
+  - recourse
+authors:
+  - name: Hangzhi Guo
+    orcid: 0009-0000-6277-9003
+    corresponding: true # (This is how to denote the corresponding author)
+    affiliation: 1 # (Multiple affiliations must be quoted)
+  - name: Xinchang Xiong
+    affiliation: 2
+  - name: Wenbo Zhang
+    affiliation: 1
+  - name: Amulya Yadav
+    orcid: 0009-0005-4638-9140
+    affiliation: 1
+affiliations:
+ - name: Penn State University, USA
+   index: 1
+ - name: Duke University, USA
+   index: 2
+date: 10 December 2023
+bibliography: paper.bib
+
+# Optional fields if submitting to a AAS journal too, see this blog post:
+# https://blog.joss.theoj.org/2018/12/a-new-collaboration-with-aas-publishing
+# aas-doi: 10.3847/xxxxx <- update this with the DOI from AAS once you know it.
+# aas-journal: Astrophysical Journal <- The name of the AAS journal.
+---
+
+# Summary
+
+Counterfactual explanation[^1] techniques provide contrast cases to individuals adversely affected by ML preictions.
+For instance, recourse methods can provide suggestions for loan applicants who have been rejected by a bank's ML algorithm, or give practical advice to teachers handling students at risk of dropping out.
+Numerous recourse explanation methods have been recently proposed.
+Yet, current research practice focuses on medium-sized datasets (typically around ~50k data points).
```

Comment:
A lot of recourse explanation takes a lot of time to run. As a reuslt, the current result practices are forced to test on medium-size datastes (as it doesn't scale up).